automation/common/deploy-kubevirt-and-cdi.sh

```diff
@@ -28,13 +28,11 @@ spec:
   configuration:
     selinuxLauncherType: "virt_launcher.process"
     developerConfiguration:
+      useEmulation: true
```

Comment:
you don't need this, when ssp tests runs on azure

Reply:
Ok, removed this change.